src/main/kotlin/project2/Network.kt

```diff
@@ -0,0 +1,57 @@
+package project2
+
+class Network(c:Int,trains:List<Train>, segmentcount:Int) {
```

Comment:
Wie in den anderen PRs vermerkt, könnt ihr Felder gleich hier deklarieren - das macht den Code deutlich kürzer und leichter zu lesen.